election-api-javascript/src/resultsService.js

```diff
@@ -16,7 +16,7 @@ function resultStore() {
     }
 
     function reset() {
-        store = [];
```

Comment:
Lets get this merged, but I think this could do with a comment as it might not be entirely clear why we are using `splice(0)` here, essentially as a workaround for an immutable (const) store.